### ISSUE TITLE
Error Message mostly in Ship causing lag

### ISSUE THREAD

**Issue by Fabimilo** [HolographicWings/LethalExpansion#73]
Hello,
I've seen the below error message when playing with friends and everytime this comes up I also see frame drops in the ship.
Don't appear everytime but often but also goes away after we finished the moon and taken off.

![Error](https://github.com/HolographicWings/LethalExpansion/assets/155649590/fe599b28-b7c2-4c9e-a9c5-f44d92c9a277)

**Comment by HolographicWings** [comment]
> Hello, I've seen the below error message when playing with friends and everytime this comes up I also see frame drops in the ship. Don't appear everytime but often but also goes away after we finished the moon and taken off.
> 
> ![Error](https://private-user-images.githubusercontent.com/155649590/294206509-fe599b28-b7c2-4c9e-a9c5-f44d92c9a277.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ0MDI4NTEsIm5iZiI6MTcwNDQwMjU1MSwicGF0aCI6Ii8xNTU2NDk1OTAvMjk0MjA2NTA5LWZlNTk5YjI4LWI3YzItNGM5ZS1hOWM1LWY0NGQ5MmM5YTI3Ny5QTkc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTA0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEwNFQyMTA5MTFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wZWFhMGMzNjdmYmUwNzRiZjhkN2ZmNzY3YzI4MDIzNTRkMjIyYTM3Y2M4YmQ3ZDE0NTU0OTA3OTk2ZWM0NDI3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.PPrReArR3BkHEG8rz0uUvxjbx4AeJvtQBfV8Bsiorcw)

The mod doesn't patch EnemyAI
it may be from another mod
they may be incompatibles

**Comment by theOneTrueFunniBoi** [comment]
@Fabimilo I've got the same issue, if we compare mods we may find the culprit.

**Comment by HolographicWings** [comment]
> @Fabimilo I've got the same issue, if we compare mods we may find the culprit.

yes, with more info i could see if the issue is from LE and fix it
